src/virus_clade_utils/cladetime.py

```diff
@@ -69,6 +74,20 @@ def __init__(self, sequence_as_of=None, tree_as_of=None):
         else:
             self.url_ncov_metadata = None
 
+    @property
+    def ncov_metadata(self):
+        return self._ncov_metadata
+
+    @ncov_metadata.getter
+    def ncov_metadata(self) -> dict:
```

Comment:
Nextstrain only began publishing their pipeline metadata at the beginning on 8/1/24, so we won't find it for earlier dates.